.gitignore

```diff
@@ -6,4 +6,5 @@
 venv/
 .pytest_cache/
 **__pycache__/
-db.sqlite3
+.sqlite3
+.db3
```

Comment:
The `.gitignore` file should not include `.db3` files as per the checklist requirement: 'Make sure you don't push db files (files with `.sqlite`, `.db3`, etc. extension)'.